osu.Game/Screens/Play/SubmittingPlayer.cs

```diff
@@ -234,6 +234,18 @@ protected override void StartGameplay()
             spectatorClient.BeginPlaying(token, GameplayState, Score);
         }
 
+        public override bool Pause()
+        {
+            bool wasPaused = GameplayClockContainer.IsPaused.Value;
+
+            bool paused = base.Pause();
+
+            if (!wasPaused && paused)
+                Score.ScoreInfo.Pauses.Add((int)Math.Round(GameplayClockContainer.CurrentTime));
```

Comment:
There's argument that we may want some degree of debounce here for sanity (100 ms or something arbitrary). Any thoughts on that?

Reply:
I'm not sure it's useful given the process of *pausing itself* has a debounce / cooldown mechanism built in already.